Content.Shared/Light/Component/SharedLightBulbComponent.cs

```diff
@@ -0,0 +1,116 @@
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Light.Component;
+
+/// <summary>
+/// Component that represents a light bulb. Can be broken, or burned, which turns them mostly useless.
+/// TODO: Breaking and burning should probably be moved to another component eventually.
+/// </summary>
+[RegisterComponent]
+public sealed class LightBulbComponent : Robust.Shared.GameObjects.Component
```

Comment:
rename the file accordingly (also VV fields)  (also doesnt this need NetworkedComponent?)